src/esp/physics/bullet/BulletRigidStage.cpp

```diff
@@ -167,5 +172,10 @@ const Magnum::Range3D BulletRigidStage::getCollisionShapeAabb() const {
   return combinedAABB;
 }  // getCollisionShapeAabb
 
+std::string BulletRigidStage::getBulletDebugName(int subpartId) {
```

Comment:
Put in BulletBase.h as pure virtual and override here please.

Reply:
Why? This signature (including subpartId) is specific to stage.

Reply:
If we add other constructs, such as articulated bullet objects, will we want this same debug functionality?

Reply:
If so, that's what BulletBase is for, setting up an interface for expected, bullet-specific, functionality we want to see in concretions.

Reply:
>If we add other constructs, such as articulated bullet objects, will we want this same debug functionality?
No. subpartId is specific to Stage.

Reply:
But getBullletDebugName is not unique to stage, is it? The methods for object and stage look nearly identical with the exception of passing an id, which can be defaulted.

Reply:
Ok, it looks like we disagree here. You, Alex, and I can discuss on Monday or Tuesday over VC.

Reply:
We chatted offline. I don't think this is appropriate to add to BulletBase. I renamed to getCollisionDebugName to make the method sound less general-purpose.